lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_generate_strings_file_from_code.rb

```diff
@@ -0,0 +1,74 @@
+require 'open3'
+
+module Fastlane
+  module Actions
+    class IosGenerateStringsFileFromCode < Action
+      def self.run(params)
+        globbed_paths = params[:paths].map { |p| File.file?(p) ? p : "#{p}/**/*.{m,swift}" }
+        files = Dir.glob(globbed_paths)
+
+        flags = [('-q' if params[:quiet]), ('-SwiftUI' if params[:swiftui])].compact
+
+        out, status = Open3.capture2e('genstrings', '-o', params[:output_dir], *flags, *files)
+
+        UI.error("genstrings failed with exit code #{status.exitstatus}") unless status.success?
+        UI.command_output(out) unless params[:quiet]
+
+        # Return the warnings as an Array
+        out.split("\n")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Generate the .strings files from your ObjC and Swift code'
+      end
+
+      def self.details
+        'Use genstrings to generate the .strings files from your ObjC and Swift code.'
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :paths,
+                                       env_name: 'FL_IOS_GENERATE_STRINGS_FILE_FROM_CODE_PATHS',
+                                       description: 'Array of directory paths to scan for `.m` and `.swift` files. The entries can contain glob patterns too.',
```

Comment:
When I run `bundle exec fastlane action ios_generate_strings_file_from_code`, I got this error:

```
[!] Do not let descriptions end with a '.', since it's used for user inputs as well for key :paths
```

<img width="1398" alt="Screen Shot 2021-11-19 at 3 46 22 pm" src="https://user-images.githubusercontent.com/1218433/142566497-b72fc302-358f-4acc-a15e-41c76213b402.png">

The command return a non 0 exit code.


```suggestion
                                       description: 'Array of directory paths to scan for `.m` and `.swift` files. The entries can contain glob patterns too',
```

Reply:
Interestingly, Fastlane doesn't complain about the final `?` in the `swiftui` parameter 😄

Reply:
Fixed in 2b4d317230f1383dcec12361441a8f0c52ce5265